samples/AzureOpenAI/Advanced_SuggestJob/app.json

```diff
@@ -1,8 +1,8 @@
 {
   "id": "a11337a1-bc23-4acd-8749-1a7ce3f9e6d9",
   "name": "Business Central Copilot Toolkit Demo",
-  "publisher": "BC 23",
-  "version": "0.9.0.0",
+  "publisher": "BC 24",
+  "version": "1.0.0.0",
```

Comment:
Minor, I would leave the version to be 0.something (e.g. 0.10.0.0), just because 1.something makes it look like this is shippable code